### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by jmorris644** [jonferreira/node-red-contrib-alexa-remote#2]
When you have time can you please add a little more docs? I would like to test this out but I don't know enough about setting up Alexa devices to do it.

Thanks

Joe

**Comment by jonferreira** [comment]
Yes I will @jmorris644 but it's fairly simple, there's not "much" you can do with it for now.

Simple retrieve your cookie with https://github.com/Apollon77/alexa-remote and then use one of the 3 nodes available for now

**Comment by MattL0** [comment]
You can get your cookie from chrome too. 

Connect on amazon web app .
right click somewhere on the page and click inspect.
Click on the network tab.
Press f5
pick one of the name in the list
click on the header tab
scroll down to cookie ( if you do not see it click on another name)
copy all what is in here and paste it in the cookie section of the node-red node. ( note host : alexa.amazon.com and paste it to ''amazon page '' field of the node-red node

**Comment by emilmanta** [comment]
hola. para amazon.es funciona? porque intento configurarlo y no lo consigo. he cambiado tambien para amazon.es y tampoco.

**Comment by emilmanta** [comment]
Hi. solved. I already have it online. but I have a question:
the alexa speak node is for me to tell her, what message does she have to answer? I explain to you: what I want is that I am turning on several light bulbs and various devices and I want that when I give the order from the mobile or wherever I also put a message. for example if I turn on a light when I give the Alexa command that says: turn on the light. I don't know if it can be done through alexa speak. and if it is possible in what way do I have to send the message so that the node reads it?
